django/thunderstore/modpacks/api/experimental/views/legacyprofile.py

```
@@ -29,6 +38,26 @@ def get_rate(self) -> str:
+
+def push_export_creation_to_kafka(key: str, file_size_bytes: int):
+    try:
+        send_kafka_message(
+            topic=KafkaTopic.A_LEGACY_PROFILE_EXPORT_V1,
+            payload_string=AnalyticsEventLegacyProfileExport(
+                id=key,
+                timestamp=timezone.now(),
+                file_size_bytes=file_size_bytes,
+            ).json()
+        )
+    except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)